File: src/main/java/io/vproxy/vfx/ui/table/VTableSharedData.java
package io.vproxy.vfx.ui.table;

public class VTableSharedData<S> {
    public final VTableView<S> tableView;
    public long rowAdder = 0;

    public VTableSharedData(VTableView<S> tableView) {
        this.tableView = tableView;
    }
}

File: src-ts/src/ui/table/VTableSharedData.ts
import type { VTableView } from './VTableView.js';

export class VTableSharedData<S> {
  public readonly tableView: VTableView<S>;
  public rowAdder: number = 0;

  constructor(tableView: VTableView<S>) {
    this.tableView = tableView;
  }
}
